### DISCUSSION TITLE
deformable DETR 실험 결과

### DISCUSSION THREAD

**Discussion by kmouleejunhyuk** [boostcampaitech2/object-detection-level2-cv-06#30] [Show and tell]
wandb: https://wandb.ai/cider6/detr/runs/rt27qvpj?workspace=user-torque
finetuned_weight: https://drive.google.com/file/d/1q3-1K2ylDgIYziuzXjmxa2w7UBvJFJRg/view?usp=sharing

+validation time까지 합치니 총 학습시간 약 9h(wandb 기준)
1e-4, stepLR at 20epoch, 30epoch
한줄결론: test set의 confidence가 낮은걸로 보아, 상이한 데이터가 많은 듯 합니다
최종제출 0.560
thr 0
![image](https://user-images.githubusercontent.com/49234207/135619097-da54de50-f5a9-45ab-8681-fc33fc6955b5.png)

thr .25
![image](https://user-images.githubusercontent.com/49234207/135619148-ffe532d8-b20b-4b8e-87d0-6125c9d94769.png)

thr .5
![image](https://user-images.githubusercontent.com/49234207/135619173-6c1a32dd-019d-40d0-89cc-8e9a49d19f4e.png)

thr .75
![image](https://user-images.githubusercontent.com/49234207/135619196-e988693b-f76f-44ad-89c5-1d68113d6e7d.png)

thr .9
![image](https://user-images.githubusercontent.com/49234207/135619228-7bb02f8b-6462-4dc7-b0da-1a5c6812bd27.png)

**Comment by jaehwan-AI** [comment]
30 epoch 학습시간은 어느 정도 걸리셨나요? (+batch size는 몇으로 하셨나요?)

**Reply by kmouleejunhyuk** [reply]
@jaehwan-AI  메모리 터져서 4 batch로 수행했고, 약 5시간 걸렸습니다

**Reply by jaehwan-AI** [reply]
오.. 모델이 상당히 무겁네요.. 참고하겠습니다
답변 감사합니다!

**Reply by kmouleejunhyuk** [reply]
@jaehwan-AI  넵 상세한 로스같은건 첨부한 wandb 참고해주세요

**Comment by jaehwan-AI** [comment]
학습 log에 보시면 bbox_mAP_50가 찍혀있을 텐데 혹시 그 값을 알 수 있을까요?
어제 마스터 클래스에서 LB와의 metric 차이를 비교해보는 작업을 했던 것 같은데 현재 성능이 가장 높은 이 모델은 얼마나 차이가 나는지 궁금해서 여쭤봅니다

**Reply by kmouleejunhyuk** [reply]
@jaehwan-AI  죄송합니다. 원본은 쥬피터 노트북으로 돌렸는데, 그때는 wandb에 그 값이 안찍혔어요. 대신 지금 돌리는 건 찍히고 있습니다. 다 돌리는 대로 공유할게요

**Reply by jaehwan-AI** [reply]
네 알겠습니다 감사합니다!

**Reply by kmouleejunhyuk** [reply]
@jaehwan-AI https://wandb.ai/cider6/detr/runs/rt27qvpj?workspace=user-torque에 공유해두었습니다
